src/AppInstallerCLICore/Commands/SettingsCommand.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "SettingsCommand.h"
+#include "Workflows/WorkflowBase.h"
+#include "Resources.h"
+#include <winget/UserSettings.h>
+
+namespace AppInstaller::CLI
+{
+    using namespace Utility::literals;
+    using namespace AppInstaller::Settings;
+
+    using namespace std::string_view_literals;
+
+    std::vector<Argument> SettingsCommand::GetArguments() const
+    {
+        return {};
+    }
+
+    Resource::LocString SettingsCommand::ShortDescription() const
+    {
+        return { Resource::String::SettingsCommandShortDescription };
+    }
+
+    Resource::LocString SettingsCommand::LongDescription() const
+    {
+        return { Resource::String::SettingsCommandLongDescription };
+    }
+
+    std::string SettingsCommand::HelpLink() const
+    {
+        return "https://aka.ms/winget-settings";
+    }
+
+    void SettingsCommand::ExecuteInternal(Execution::Context& context) const
+    {
+        // Show warnings only when the setting command is executed.
```

Comment:
>// Show warnings only when the setting command is executed. [](start = 8, length = 59)

You might want to have some text beforehand that will set up the warnings that are about to come out, rather than just showing them.